### ISSUE TITLE
Video?

### ISSUE THREAD

**Issue by skmar** [azgs/azwinemap#17]
Hi @jessica-azgs @Lbookman,

I'm not sure if this has been already addressed...but do we have the capabilities to upload a video to the wine database and have it play on the map?

--Stephanie

@ridavis @fmichaelconway

**Comment by jessicagood** [comment]
@skmar No, you can't add videos. This would take some work to add in but it should be possible. Is this something you need?

**Comment by skmar** [comment]
It's not something we need but something that would be nice to have. I know it's hard to estimate something like this...but do you have a rough idea of how much time it would take? Do you guys have time in your schedule to work on something like this?
